wayland-backend/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Additions
 - client: Implement `Eq` for `Backend`
 
+#### Bugfixes
+- backend: Fix error/segfault if object argument is no longer alive
```

Comment:
```suggestion
- sys/client: Fix error/segfault if object argument is no longer alive
```

BTW, that makes me realize, it's likely there is a similar bug server-side, that'd be better to fix both in the same PR is that is the case.